README.md

```diff
@@ -16,6 +16,7 @@ This project and the following release notes have been migrated from the old Ara
 
 Release | Notes
 --------|--------
+[v5.0.0](https://github.com/ArasLabs/my-recent-items/releases/tag/v5.0.0) | Bugfix and method optomization. Readme History Updated.
```

Comment:
What's the method optimization? Is this PR missing a commit(s)?
```suggestion
[v5.0.0](https://github.com/ArasLabs/my-recent-items/releases/tag/v5.0.0) | Bugfix and method optimization. Readme History Updated.
```

Reply:
Method optimization was Angela's PR. I'm combining my bugfix with her PR in a single release, this 5.0 since there was a code change

Reply:
Ah! Makes sense. Just fix the spelling typo then and this is good to go.